packages/@okta/vuepress-site/docs/api/resources/oidc/index.md

```diff
@@ -451,52 +407,39 @@ Content-Type: application/json;charset=UTF-8
-| client_assertion        | Required if the `client_assertion_type` is specified. Contains the JWT signed with the `client_secret`.[JWT Details](#token-authentication-methods)                                                                                                                               | String        |
-| client_assertion_type   | Indicates a JWT is being used to authenticate the client. Per the [Client Authentication spec](http://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication), the valid value is `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`.                           | String        |
-| token                   | An access or refresh token.                                                                                                                                                                                                                                                       | String        |
-| token_type_hint         | A hint of the type of `token`. Valid values are `access_token` and `refresh_token`.                                                                                                                                                                                               | String (Enum) |
-
-> Native applications should not provide -- and by default do not store -- `client_secret` (see [Section 5.3.1 of the OAuth 2.0 spec](https://tools.ietf.org/html/rfc6819#section-5.3.1)). They can omit `client_secret` from the above request parameters when revoking a token.
+> Note: The `/revoke` endpoint requires token authentication. See the [Client Authentication Methods](#client-authentication-methods) section for more information on which method to choose and how to use the parameters in your request.
 
+| Parameter               | Description                                                                                       | Type          |
+| :---------------------- | :------------------------------------------------------------------------------------------------ | :-----        |
+| token                   | An access or refresh token.                                                                       | String        |
+| token_type_hint         | A hint of the type of `token`. Valid values are `access_token` and `refresh_token`.               | String (Enum) |
 
-For more information about token authentication, see [Token Authentication Methods](#token-authentication-methods).
+> Native applications should not provide -- and by default do not store -- the `client_secret` (see [Section 5.3.1 of the OAuth 2.0 spec](https://tools.ietf.org/html/rfc6819#section-5.3.1)). They can omit the `client_secret` from the above request parameters when revoking a token.
```

Comment:
It's now odd to be referring to `client_secret` when it's not mentioned in the request parameters table. Maybe this belongs down in the overall ClientAuthenticationMechanism table?

Reply:
good point